statuslight_control/lib/statuslight_control/application.ex

```diff
@@ -8,7 +8,7 @@ defmodule StatuslightControl.Application do
   def start(_type, _args) do
     children = [
       # Starts a worker by calling: StatuslightControl.Worker.start_link(arg)
-      {StatuslightControl.Statuslight, Application.get_env(:statuslight_control, :renderer)}
+      {StatuslightControl.Statuslight, []}
```

Comment:
It's better to store this as a module attribute vs in the state of the Genserver